app-i18n/ibus-libpinyin/ibus-libpinyin-1.11.1.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+PYTHON_COMPAT=( python{2_7,3_5,3_6} )
+PYTHON_REQ_USE="sqlite(+)"
```

Comment:
This is wrong. Grepping for sqlite in unpacked tarball shows that Python's `sqlite3` module, but `dev-db/sqlite` is directly used at build time and run time. If `EAPI="7"` was used, then `dev-db/sqlite` should be in `BDEPEND`, `DEPEND` and `RDEPEND`.

Reply:
i have updated to EAPI 7 , and done what you said *except* i haven't added sqlite3 to the BDEPEND - it is my understanding that it is only appropriate for the DEPEND and RDEPEND.

Reply:
https://github.com/libpinyin/ibus-libpinyin/blob/dcd9f98ae004cfb1270d472adc9cafcca0e625da/configure.ac#L60-L62 requires sqlite in DEPEND.

And https://github.com/libpinyin/ibus-libpinyin/blob/dcd9f98ae004cfb1270d472adc9cafcca0e625da/configure.ac#L64-L67 requires sqlite in BDEPEND.
https://github.com/libpinyin/ibus-libpinyin/blob/75806735002f940927648b5abd04a4b6704518fc/data/Makefile.am#L48 and https://github.com/libpinyin/ibus-libpinyin/blob/75806735002f940927648b5abd04a4b6704518fc/data/Makefile.am#L54 call `/usr/bin/sqlite3` executable (BDEPEND).

Reply:
Hmm, OK. you are right, i am wrong. After a second look am persuaded - I knew it was there in the build scripts but my thinking was that the makefile was actually intending to create a database on the target and should be using the target sqlite. This isn't right though, as it doesn't matter if this is created on the host sqlite as it is just a db file, and is platform agnostic. As such it can be thought of as a build tool and it should (as you say) go into BDEPEND. I'll do this in separate pull request (I will close this one as it is now stale as the version bump has been merged but not the stabilization etc - which seems like it needs a separate pull request). thanks